sqlmesh/core/snapshot/evaluator.py

```diff
@@ -535,9 +537,9 @@ def _create_snapshot(
             return
 
         parent_snapshots_by_name = {
-            snapshots[p_sid].name: snapshots[p_sid] for p_sid in snapshot.parents
+            snapshots[p_sid].fqn: snapshots[p_sid] for p_sid in snapshot.parents
```

Comment:
Please make sure to update https://github.com/TobikoData/sqlmesh/blob/87f37b04ed416d53cc084bbf03357e82e3302c7b/sqlmesh/core/snapshot/evaluator.py#L335 too